.github/workflows/test_and_deploy.yml

```diff
@@ -48,15 +37,13 @@ jobs:
       - name: Build
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
-        run: npx lerna run build
+        run: npx lerna run --ignore orbit.kiwi build
```

Comment:
This does not leave out the docs build. I think using `@kiwicom/orbit.kiwi` would work, but haven't checked it.